.github/workflows/nightly.yml

```diff
@@ -115,6 +115,18 @@ jobs:
       E2E_API_KEY: ${{ secrets.AC_PROD_E2E_API_KEY }}
       E2E_LICENSE_KEY: ${{ secrets.AC_PROD_E2E_LICENSE_KEY }}
 
+  k8s-e2e-tests-latest-flux:
+    uses: ./.github/workflows/component_k8s_e2e.yml
+    with:
+      scenarios: '["apm", "collector", "fleet-control", "ebpf", "dynamic"]'
+      use_latest_flux: true
+    secrets:
+      NR_SYSTEM_IDENTITY_CLIENT_ID: ${{ secrets.AC_PROD_E2E_NR_SYSTEM_IDENTITY_CLIENT_ID }}
+      NR_SYSTEM_IDENTITY_PRIVATE_KEY: ${{ secrets.AC_PROD_E2E_NR_SYSTEM_IDENTITY_PRIVATE_KEY }}
+      E2E_ACCOUNT_ID: ${{ secrets.AC_PROD_E2E_ACCOUNT_ID }}
+      E2E_API_KEY: ${{ secrets.AC_PROD_E2E_API_KEY }}
+      E2E_LICENSE_KEY: ${{ secrets.AC_PROD_E2E_LICENSE_KEY }}
```

Comment:
I would not executute all of them again, I would execute everything pointing to latest maybe

Reply:
But then we would stop executing them pointing to the version currently used, wouldn't we? 🤔 

What if we executed all of then again pointing to latest in a different schedule? (less frequently maybe)

Reply:
also that would work for me

Reply:
> But then we would stop executing them pointing to the version currently used, wouldn't we? 🤔
we do it already often in Prs, no?